### PR TITLE
Wigi: improve support for RTL

### PR DESCRIPTION
**Pull request by ykuzmenko73**
**In the branch**:
image

image

image

**in the master**:
image
image

image

### CODE REVIEW

**Comment by ykuzmenko73** [comment]
[Wrong text alignment in expanders in Arabic](https://trello.com/c/o386YsdH/6016-wrong-text-alignment-in-expanders-in-arabic)

File: lib/tropic/tropic2form.flow [hunk 1]
```diff
@@ -309,10 +309,14 @@ tropic2Acc(b : Tropic, parent : TParentInfo, sheet : Stylesheet, metricsOnly : b
 		TDynamicGroup2(stackChanges, currentStack, combiner) : {
 			fgroupChanges : DynamicBehaviour<[FGroupChange]> = make([]);
 			tropicItems : DynamicBehaviour<[TDynamicGroupItem]> = make([]);
-			verticalCombiner = if (combiner == TLinesCombiner()) FAddition() else FMax();
-			rtl = combiner == TColsACombiner() && getDefaultRtl();
+			verticalCombiner = if (combiner == TLinesCombiner() || combiner == TLinesACombiner()) FAddition() else FMax();
+			rtl = getDefaultRtl();
```
Comment (brawaga) [inline lib/tropic/tropic2form.flow]: I do not remember exactly what for, but surely this condition (rtl) was added for a reason. Did you ensure all combiners work correctly?

**Comment by brawaga** [comment]
imageExcellent.

File: lib/form/gui.flow [hunk 2]
```diff
@@ -490,6 +492,42 @@ Lines(rows) {
...
+			Constructor(
+				Translate(
+					diffWidth,
+					make(0.0),
+					Inspect([Width(widths[i])], l)
+				),
+				\ -> {
+					uns = connectSelect2Distinctu(maxWidthB.first, widths[i], diffWidth, \maxWidth, width -> maxWidth - width);
+					\ -> {
+						if (i == 0) maxWidthB.second();
```
Comment (yegordovganich) [inline lib/form/gui.flow]: only if i == 0?

Comment (ykuzmenko73) [reply lib/form/gui.flow]: maxWidthB is "global" behavior, we should destroy it only once.
there is another option of course, I could pass it to the Lines level and wrap resulting Lines with another Constructor.

Comment (ykuzmenko73) [reply lib/form/gui.flow]: rewritten to be less obstructive

**Comment by artxp** [comment]
@ykuzmenko73 did you test drop down. I remember initial fix affected it behaviour.

**Pull request merged**